core-libs/setup/ssr/error-handling/express-error-handlers/express-error-handlers.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { CmsPageNotFoundOutboundHttpError } from '@spartacus/core';
+import { defaultExpressErrorHandlers } from './express-error-handlers';
+
+describe('errorResponseHandlers', () => {
```

Comment:
[MINOR]
```suggestion
describe('defaultExpressErrorHandlers', () => {
```

Reply:
Then it should be not `errorResponseHandlers` but `expressErrorHandlers`, I guess?